### PR TITLE
Enhance errors summary log to session highlights log (fix #692)

### LINKED ISSUE #692: Enhance errors summary log to session highlights log
The configuration path should support both (old) `log.errors_subpath` and (new) `log.highlights_subpath`.

### PR DESCRIPTION
**Pull request by ayalash**

### CODE REVIEW

**Comment by ayalash** [comment]
@vmalloc Please change the "The Error Log" section of doc/logging.rst according to this PR, as we discussed f2f.

File: slash/conf.py [hunk 1]
```diff
@@ -51,6 +51,7 @@
         "subpath": "{context.session.id}/{context.test_id}/debug.log" // Doc("Path to write logs to under the root"),
         "session_subpath": "{context.session.id}/session.log",
         "errors_subpath": None // Doc("If set, this path will be used to record errors added in the session and/or tests"),
```
Comment (vmalloc) [inline slash/conf.py]: Add a deprecation notice here pointing to the new config value (in the doc string)

File: slash/core/result.py [hunk 2]
```diff
@@ -227,7 +227,7 @@ def _add_error(self, error_list, error=None, frame_correction=0, exc_info=None,
             if is_failure:
                 # force the error object to be marked as failure
                 error.mark_as_failure()
-            _logger.debug('Error added: {0}\n{0.traceback}', error, extra={'to_error_log': 1})
+            _logger.debug('Error added: {0}\n{0.traceback}', error, extra={'highlights': 1})
```
Comment (vmalloc) [inline slash/core/result.py]: I prefer `highlight`, not `highlights` because the log itself is a highlight (singular)

**Comment by ayalash** [comment]
@vmalloc I pushed a commit with the changes. Please review.

**Pull request merged**